packages/puppeteer-core/src/node/ChromeLauncher.ts

```diff
@@ -266,3 +272,26 @@ function convertPuppeteerChannelToBrowsersChannel(
       return BrowsersChromeReleaseChannel.CANARY;
   }
 }
+
+/**
+ * Extracts all features from the given command-line flag
+ * (e.g. `--enable-features`, `--enable-features=`).
+ *
+ * Example input:
+ * ["--enable-features=NetworkService,NetworkServiceInProcess", "--enable-features=Foo"]
+ *
+ * Example output:
+ * ["NetworkService", "NetworkServiceInProcess", "Foo"]
+ *
+ * @internal
+ */
+export function getFeatures(flag: string, options: string[] = []): string[] {
+  return options
+    .filter(s => {
+      return s.startsWith(flag.endsWith('=') ? flag : `${flag}=`);
+    })
+    .map(s => {
+      return s.split(new RegExp(`${flag}` + '=\\s*'))[1]?.trim();
+    });
+  return opts.length === 0 ? [] : opts.join(',').split(',');
```

Comment:
```suggestion
```

Reply:
The first suggestion was actually wrong. `ops` was created as an intermediate value. I was having compilation issues otherwise